pkg/storage/SQL/plan_favorite_create.sql

```diff
@@ -0,0 +1,26 @@
+INSERT INTO plan_favorite(
+    id,
+    model_plan_id,
+    user_id,
+    created_by,
+    created_dts,
+    modified_by,
+    modified_dts
+)
+VALUES (
+    :id,
+    :model_plan_id,
+    :user_id,
+    :created_by,
+    :created_dts,
```

Comment:
I think the issue [Tom brings up here](https://github.com/CMSgov/mint-app/pull/190#issuecomment-1230986948) is from this line of code.

Most (all?) of the other SQL files that create entities in our DB don't take `created_dts` or `modified_dts` as parameters, they just fall back on the defaults (which are `CURRENT_TIMESTAMP` and `NULL`, respectively), which I think makes sense here too.

Reply:
@ClayBenson94, @OddTomBrooks , thanks for catching this. That is definitely the issue, and it should be using CURENT_tIMESTAMP. Great find!